CHANGELOG.md

```diff
@@ -50,12 +44,45 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   existing caching mechanisms (e.g., supported by DCAP).
   See `openenclave/attestation/tdx/evidence.h` for more detail of the APIs.
 
-## Changed
-- Fix the incorrect behavior of pthread_mutex_init() and std::mutex such that they no longer create a recursive lock by default.
-- `oe_verify_report` now reports the verification result in the `parsed_report` output (only valid when the API returns OE_OK or OE_TCB_LEVEL_INVALID). For the SGX case, the definition of the result is based on `oe_sgx_tcb_status_t`.
+- Added support for V4 collateral from Intel for SGX and TDX
+
+### Changed
+- snmalloc (0.6.0) now only requires 16KiB initially per thread, compared to 256 KiB previously (0.5.3). Thread-local heaps only grow by a constant amount until the next power of 2, eg. 16KiB, 16KiB, 32KiB, 64KiB, ... 1MiB, 2MiB, 2MiB, ..., as opposed to a fixed amount previously (256 KiB). Allocator metadata is kept separately, making corruption attacks less likely. There are some applications for which this update in snmalloc will result in an increase in EPC memory as specified in the enclave configuration file.
+
```

Comment:
- Open Enclave SDK is no longer tested on Ubuntu 18.04. Ubuntu 18.04 is at EOL. v0.19.0 will not contain packages for Ubuntu 18.04.